Teacher Workout/Features/Authentication/SignIn/Coordinator/SignInCoordinator.swift

```diff
@@ -28,4 +29,10 @@ extension SignInCoordinator: SignInViewDelegate {
         navigationController.dismiss(animated: true)
         delegate?.signInCoordinatorDidClose(self)
     }
+
+    func signInViewDidTapSignIn(_: SignInView) {
+        navigationController.dismiss(animated: true)
+        delegate?.signInCoordinatorDidClose(self)
```

Comment:
@vladstanescu94 I think we should call just one delegate here, `signInCoordinatorDidTapSignIn`. And the delegate to do work is needed it.